src/org/usfirst/frc/team2175/subsystem/ShooterSubsystem.java

```diff
@@ -37,15 +36,13 @@ public ShooterSubsystem() {
 
         shooterMotor = new CANTalon(
                 wiringProperties.getShooterShooterMotorDeviceNumber());
-        shooterMotor.setFeedbackDevice(FeedbackDevice.QuadEncoder);
-        shooterMotor.reverseSensor(true);
-        shooterMotor.changeControlMode(TalonControlMode.Speed);
-        shooterMotor.setFeedbackDevice(FeedbackDevice.QuadEncoder);
-        shooterMotor.setProfile(0);
+        // shooterMotor.setFeedbackDevice(FeedbackDevice.QuadEncoder);
+        // shooterMotor.reverseSensor(true);
+        // shooterMotor.changeControlMode(TalonControlMode.Speed);
+        // shooterMotor.setFeedbackDevice(FeedbackDevice.QuadEncoder);
+        // shooterMotor.setProfile(0);
```

Comment:
It would be nice to have a method in the shooter subsystem that switches between closed-loop and open-loop mode.